Tests/SQLTests/DataQueryTests.swift

```diff
@@ -2,6 +2,21 @@ import SQL
 import XCTest
 
 final class DataQueryTests: XCTestCase {
+    
+    static let allTests = [
```

Comment:
Can you move this to the bottom of the file? That will match the rest of the tests in Vapor packages.

Reply:
I was actually hoping you would agree to have it on the top ... it's just too easy to forget to add the tests for linux and without testing we got all tests I hoped having it on the top always would help?